tools/deployment_tool/utils.js

```diff
@@ -154,7 +155,8 @@ const deployContracts = async (
             gas = await web3.eth.estimateGas(deploymentObject.transactionObject);
         }
         if (!gasPrice) {
-            gasPrice = await web3.eth.getGasPrice();
+            gasPrice = new BN(await web3.eth.getGasPrice());
+            gasPrice = gasPrice.muln(10).toString(10);
```

Comment:
I didn't understand this part 🤔

Reply:
I deleted it. We had to increase the gas price for testing in order for our transactions to be mined timely.